README.md

```diff
@@ -1,58 +1,144 @@
-`
-
-See [ART_NOTES.md](Documentation/ART_NOTES.md) for more information on the art of Boss Room.
+`. 
+  
+For more information on the art of Boss Room, see [ART_NOTES.md](Documentation/ART_NOTES.md).
 
-> __IMPORTANT__:
-> - Boss Room has been developed and tested on these Platforms (Windows, Mac, iOS, and Android).
->     - Tested on iPhone 6 and Pixel 3.
-> - Boss Room is compatible with the latest Unity LTS version.
-> - Make sure to include standalone support for Windows/Mac in your installation. 
 
 ![](Documentation/Images/Boss.png)
+  
+
+## Getting the project
+### Direct download
+ - You can download the latest version of Boss Room from our [Releases](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/releases) page. 
+ - __Alternatively:__ click the green `Code` button and then select the 'Download Zip' option.  Please note that this will download the branch that you are currently viewing on Github.  
+ - __Windows users:__ Using Windows' built-in extraction tool may generate an "Error 0x80010135: Path too long" error window which can invalidate the extraction process. A workaround for this is to shorten the zip file to a single character (eg. "c.zip") and move it to the shortest path on your computer (most often right at C:\\) and retry. If that solution fails, another workaround is to extract the downloaded zip file using [7zip](https://www.7-zip.org/).
+
+
+### Installing Git LFS to clone locally
+
+Boss Room uses Git Large Files Support (LFS) to handle all large assets required locally. See [Git LFS installation options](https://github.com/git-lfs/git-lfs/wiki/Installation) for Windows and Mac instructions.
+
+## Registering the project with Unity Gaming Services (UGS)
+
+Boss Room leverages several services from UGS to facilitate connectivity between players. To use these services inside your project, you must [create an organization](https://support.unity.com/hc/en-us/articles/208592876-How-do-I-create-a-new-Organization-) inside the Unity Dashboard, and enable the [Relay](https://docs.unity.com/relay/get-started.html) and [Lobby](https://docs.unity.com/lobby/game-lobby-sample.html) services. Otherwise, you can still use Boss Room without UGS.
+
+## Opening the project for the first time
+
+Once you have downloaded the project, follow the steps below to get up and running:
+ - Check that you have installed the most recent [LTS editor version](https://unity.com/releases/2021-lts).
+ 	- Include standalone support for Windows/Mac in your installation. 
+ - Add the project to the _Unity Hub_ by clicking on the **Add** button and pointing it to the root folder of the downloaded project.
+ 	- __Please note :__ the first time you open the project Unity will import all assets, which will take longer than usual.
+
+ - Once open, select the 'Startup' scene and click the **Play** button. You can then host a new game or join an existing one using the in-game UI.
```

Comment:
This isn't needed anymore. We have an automation that starts the game with Startup first, whatever the scene you're in.